README.markdown

```diff
@@ -34,6 +34,21 @@ Configuration
 	- office_hours_contact_info: displays next to the office hours section on the home page when logged in
 	- after_hours_contact_info: displays next to the after hours section on the home page when logged in
 
+Setup Python Environment
+-------
+virtualenv postmaster
+cd postmaster/
+git clone git@github.com:UCF/PostMaster.git src
+source bin/activate
+cd src/
+pip install -r requirements.txt
+deactivate
```

Comment:
Lines 45 and 46 could probably be removed here (unless there's a particular reason to re-activate the virtualenv @jmbarne3)

Reply:
This is probably a bug specific to this particular version of virtualenv running on osx. I would remove the reactivation steps. Potentially you could add a note about the bug.